src/actions/actionsFactory.ts

```diff
@@ -1,37 +1,39 @@
 import { Uri } from "vscode";
 import { IAction, ICommandAction, ActionType, IFileAction, ActionJsonKey } from './interfaces';
 import { CommandAction, FileAction } from './impl';
-import _ = require("lodash");
+import { isNil, isEmpty} from "lodash";
 
 export class ActionsFactory {
     public static createAction(jsonAction: any) : IAction {
         const actionType = jsonAction[ActionJsonKey.ActionType];
-        if (_.isEmpty(actionType)) {
+        if (isEmpty(actionType)) {
             throw new Error(`${ActionJsonKey.ActionType} is missing`);
         }
         switch (actionType) {
-            case ActionType.Command:
-                const commandAction: ICommandAction = new CommandAction();
-                const commandName = jsonAction[ActionJsonKey.CommandName];
-                if (!_.isNil(commandName)) {
-                    commandAction.name = commandName;
-                } else {
-                    throw new Error(`${ActionJsonKey.CommandName} is missing for actionType=${actionType}`);
+            case ActionType.Command: {
+                    const commandAction: ICommandAction = new CommandAction();
+                    const commandName = jsonAction[ActionJsonKey.CommandName];
+                    if (!isNil(commandName)) {
```

Comment:
```suggestion
                    if (!commandName) {
```
I think that better because in your approach the empty '' case isn't catch

Reply:
Done